Monika After Story/game/definitions.rpy

```diff
@@ -4490,6 +4490,15 @@ init -1 python:
         return persistent._mas_sunrise <= _curr_minutes < persistent._mas_sunset
 
 
+    def mas_isSunnyNow():
```

Comment:
I think we may want to change the names of these functions, as at a glance they come across to account for weather when in reality they don't.

Ideally this should be `mas_isSunUpNow()` and `mas_isSunny()` changes to `mas_isSunUp()`

This way there won't be any potential misunderstandings/misuses of the function